x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -175,8 +174,8 @@ export const navigationCategories: NavigationCategories = [
       SecurityPageName.policies,
       SecurityPageName.trustedApps,
       SecurityPageName.eventFilters,
-      SecurityPageName.blocklist,
       SecurityPageName.hostIsolationExceptions,
+      SecurityPageName.blocklist,
```

Comment:
Thanks for sorting this!

Also, seems the hostIsolationException item is not disabled when env does not match the criteria. We already have this logic in the existing navigation [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/navigation/use_security_solution_navigation/use_navigation_items.tsx#L119-L121)

Reply:
Excluded the link using the hook here: 
https://github.com/elastic/kibana/pull/132210/files#diff-d1acbf8273add55b64dd3840a1e7eca4a36c4baa3f4e5b31b86d73bd93cecd0fR143-R148

We should change this approach at some point, filtering all links when the plugin mounts.